streams/test-utils/src/test/java/org/apache/kafka/streams/TopologyTestDriverTest.java

```diff
@@ -1092,17 +1092,19 @@ public void shouldCreateStateDirectoryForStatefulTopology() {
     }
 
     @Test
-    public void shouldReturnIteratorForMultipleRecords() {
+    public void shouldReturnIterableForMultipleRecords() {
         testDriver = new TopologyTestDriver(setupSourceSinkTopology(), config);
-
-        testDriver.pipeInput(consumerRecord1);
+        final byte[] key1 = "key1".getBytes();
+        final byte[] value1 = "value1".getBytes();
+        final ConsumerRecord<byte[], byte[]> consumerRecord1 = consumerRecordFactory.create(SOURCE_TOPIC_1, key1, value1, new RecordHeaders(), 0L);
```

Comment:
nit: we usually try to keep lines under 120 characters.

Reply:
Done, my IDE settings were off regarding wrapping